src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
I think we're missing some tests here:
`applies open prop to Radix tooltip root`, `applies defaultOpen prop to Radix tooltip root`, `applies onOpenChange prop to Radix tooltip root`, `applies side prop to Radix tooltip content`, `applies align prop to Radix tooltip content`.

Bit of a weird set of tests because generally we wouldn't write tests for implementation details, but for these tests, we want to know that we're configuring the Radix primitives correctly.